.github/workflows/benchmark.yml

```diff
@@ -15,10 +15,10 @@ jobs:
         scenario: ["inference-rest.js", "inference-grpc.js"]
     steps:
       - uses: actions/checkout@v4
-      - name: Set up Python 3.10
```

Comment:
lets keep the CI at python `3.10` as it is related to the main docker python version, and just add support for 3.11 and 3.12 if thats possible.